docs/proposal.md

```diff
@@ -0,0 +1,108 @@
+Containerd CRI Integration
+=============
+Author: Lantao Liu (@random-liu)
+## Abstract
+This proposal aims to integrate [containerd](https://github.com/containerd/containerd) with Kubelet against the [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
+## Background
+Containerd is a core container runtime, which provides the minimum set of functionalities to manage the complete container lifecycle of its host system, including container execution and supervision, image distribution and storage, etc.
+
+Containerd was previously part of Docker, [introduced in Docker 1.11](https://blog.docker.com/2016/04/docker-engine-1-11-runc/), used to manage [runC](https://runc.io/) containers on the node. As is shown below, it creates a containerd-shim for each container, and the shim manages the lifecycle of its corresponding container.
+![containerd](./containerd.png)
+
+In Dec. 2016, Docker spun it out into a standalone component, and donated it to [CNCF](https://www.cncf.io/) in Mar. 2017.
```

Comment:
Seems like we should use `Docker Inc.` here? It is confusing between Docker & Docker Inc in this part.